packages/cozy-client/src/CozyClient.js

```diff
@@ -748,6 +748,13 @@ client.query(Q('io.cozy.bills'))`)
     try {
       this.dispatch(loadQuery(queryId))
       const response = await this.requestQuery(queryDefinition)
+      if (response.execution_stats) {
+        console.log(
+          `Execution stats for query ${queryId}: ${JSON.stringify(
+            response.execution_stats
+          )}`
+        )
+      }
```

Comment:
This is particular to the stack client right ? I don't think the log should be here. I wonder if it would be better suited to put this in the CozyStackLink ou CozyStackClient ?

Reply:
I prefered to avoid CozyStackClient as it would need to add the log for all find methods. 
It is for now particular to the stack client, because pouch does not implement execution_stats, so CozyStackLink makes sense, but I assumed it could be the case in the future, hence the log here, as it could be relevant for any link

Reply:
OK.